lib/ansible/cli/doc.py

```diff
@@ -369,18 +382,6 @@ def format_plugin_doc(plugin, plugin_type, doc, plainexamples, returndocs, metad
         doc['returndocs'] = returndocs
         doc['metadata'] = metadata
 
-        # generate extra data
-        if plugin_type == 'module':
-            # is there corresponding action plugin?
-            if plugin in action_loader:
-                doc['action'] = True
-            else:
-                doc['action'] = False
-
-        doc['now_date'] = datetime.date.today().strftime('%Y-%m-%d')
```

Comment:
why are you removing these?

Reply:
You mean `now_date` and `docuri`? Because they are not used at all. In this code-path, they are only passed to `get_snippet_text` and `get_man_text`:
1. `get_snippet_text` uses only `short_description`, `module` and `options`.
2. `get_man_text` explicitly ignores them as they are in `DocCLI.IGNORE`.

(Besides that, what the code does with `docuri` is strange and will probably fail if that key is ever encountered, since `doc[plugin_type]` usually results in a `KeyError` nowadays.)